.github/workflows/test-coverage.yml

```
@@ -45,5 +52,7 @@
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         with:
           projectBaseDir: backend
-          sonar.organization: learnathon-by-geeky-solutions
-          sonar.projectKey: Learnathon-By-Geeky-Solutions_momentum
+          args: >
+            -Dsonar.python.coverage.reportPaths=coverage.xml
+            -Dsonar.sources=.
+            -Dsonar.python.version=3.11
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)